asterisk_click2dial/asterisk_click2dial.py

```diff
@@ -170,7 +170,7 @@ def _reformat_number(
         assert(country_code), 'Missing country on company'
         _logger.debug('Country code = %s' % country_code)
         to_dial_number = phonenumbers.format_out_of_country_calling_number(
-            parsed_num, country_code.upper()).replace(' ', '').replace('-', '')
+            parsed_num, country_code.upper()).replace(' ', '').replace('-', '').replace('.','').replace('(', '').replace(')','')
```

Comment:
Maybe leave only numbers

--- asterisk_click2dial.py.orig	2015-02-04 09:57:16.165036583 -0500
+++ asterisk_click2dial.py	2015-02-04 09:57:51.655088636 -0500
@@ -24,6 +24,7 @@
 import logging
 # Lib for phone number reformating -> pip install phonenumbers
 import phonenumbers
+import re
 
 try:
     # Lib py-asterisk from http://code.google.com/p/py-asterisk/
@@ -169,8 +170,8 @@
         country_code = ast_server.company_id.country_id.code
         assert(country_code), 'Missing country on company'
         _logger.debug('Country code = %s' % country_code)
-        to_dial_number = phonenumbers.format_out_of_country_calling_number(
-            parsed_num, country_code.upper()).replace(' ', '').replace('-', '').replace('(', '').replace(')', '')
+	to_dial_number = re.sub("[^0-9]", "", phonenumbers.format_out_of_country_calling_number(
+	    parsed_num, country_code.upper()))
         # Add 'out prefix' to all numbers
         if ast_server.out_prefix:
             _logger.debug('Out prefix = %s' % ast_server.out_prefix)